infrastructure/ecs.tf

```diff
@@ -12,10 +12,10 @@ locals {
     "USE_SAGEMAKER_LLM"                    = local.secret_env_vars.USE_SAGEMAKER_LLM,
     "SENTRY_DSN"                           = local.secret_env_vars.SENTRY_DSN,
     "AWS_REGION"                           = local.secret_env_vars.AWS_REGION,
-    "DB_NAME" : "${module.postgres.db_instance_name}",
-    "DB_USER" : "${module.postgres.rds_instance_username}",
-    "DB_PASSWORD" : "${module.postgres.rds_instance_db_password}",
-    "DB_HOST" : "${module.postgres.db_instance_address}",
```

Comment:
I think we can skip all these bits and pieces and also do away with the `PRODUCTION_DEPLOYMENT` flag, and simply pass `DATABASE_URL` — ie the fully-qualified postgres url stored in `rds_fqdn` (assuming that's updated!) . `env.db()` in `settings/base.py` will then do the right thing. (This is how it works locally).